client/src/components/shared/styledComponents.js

```diff
@@ -0,0 +1,111 @@
+    padding: .375rem .75rem;
+    border: 1px solid #56658a;
+    border-radius: .25rem;
+    background-color: #56658a;
+    color: white;
+    transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+    font-size: ${(props) => props.fontSize}; //.875rem;
+  &:hover{
+    background-color:rgb(60, 70, 96);
+  }
+`;
+
+// export const SecondaryButton = styled.button` // Outline-Gold
+//     padding: .375rem .75rem;
+//     border: 1px solid #AB9671;
+//     border-radius: .25rem;
+//     background-color: inherit;
+//     color: #AB9671;
+//     transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+//     font-size: ${(props) => props.fontSize}; //.875rem;
+//   &:hover{
+//     background-color:rgb(119, 105, 79);
+//     color: white;
+//   }
+// `;
+
+// export const PrimaryButton = styled.button` // Gold
+//     padding: .375rem .75rem;
+//     border: 1px solid #AB9671;
+//     border-radius: .25rem;
+//     background-color: #AB9671;
+//     color: white;
+//     transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+//     font-size: ${(props) => props.fontSize}; //.875rem;
+//   &:hover{
+//     background-color: rgb(119, 105, 79);
+//   }
+// `;
+
+export const SecondaryButton = styled.button` // Outline-Purple
+    padding: .375rem .75rem;
+    border: 1px solid #56658a;
+    border-radius: .25rem;
+    background-color: transparent;
+    color: #56658a;
+    transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+    font-size: ${(props) => props.fontSize}; //.875rem;
+  &:hover{
+    background-color:rgb(60, 70, 96);
+    color: white;
+  }
+`;
+
+export const DangerButton = styled.button`
+    padding: .375rem .75rem;
+    border-radius: .25rem;
+    border: 1px solid #dc3545;
+    background-color: inherit;
+    color: #dc3545;
+    transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+  &:hover{
+    background-color: #dc3545;
+    color: white;
+  }
+`;
+
+export const OptionEditButton = styled.button` // FIXXXXXX
+    padding: .375rem .75rem;
+    border: 1px solid rgb(60, 70, 96);
+    border-radius: .25rem;
+    background-color: rgb(88, 102, 138);
+    color: white;
+    transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+    font-size: ${(props) => props.fontSize}; //.875rem;
+  &:hover{
+    background-color: rgb(60, 70, 96);
+    color: white;
+  }
+`;
+
+export const OptionDeleteButton = styled.button` // Outline-Purple
+    padding: .375rem .75rem;
+    border: 1px solid #56658a;
+    border-radius: .25rem;
+    background-color: #AB9671;
```

Comment:
same here, may be better to use red for delete button

Reply:
After merging your UI changes from main, I reckon it looks a lot better if the option's delete button is gold as it matches with the sidebar (rather than red which is pre jarring). What does everyone else think?
![image](https://user-images.githubusercontent.com/80874259/134290715-c2b3cb42-2f0e-4af7-8072-6e3ec2afe8dc.png)

Reply:
Yeah tbh I prefer this gold colour for that delete icon over having it red as well.

Reply:
Changed the delete button to red for consistency with the other pages but will still keep the style for the gold button commented out incase we wish to use it later.